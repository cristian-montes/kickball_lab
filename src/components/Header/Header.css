.header{
    display: flex;
    flex-direction: row;
   
}

.navLinks{
    margin: 5px;
    color:aqua
}

.colorClick{
    color:red
}