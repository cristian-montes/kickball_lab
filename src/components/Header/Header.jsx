import { NavLink } from 'react-router-dom';
import './Header.css';

function Header(){
  return (
    <div className='header'>
      <NavLink className='navLinks' activeClassName='colorClick' to='/' exact>
        <p>Home</p> 
      </NavLink>
      <NavLink className='navLinks' to='/teams' exact>
        <p>Teams</p>
      </NavLink>
      <NavLink className='navLinks' to='/players'>
        <p>Player</p>
      </NavLink>
     
    </div>
  );
}

export default Header;