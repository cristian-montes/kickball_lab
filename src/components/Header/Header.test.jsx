import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Header from './Header';

it('should render the header with navLinks', ()=>{
  const { container } = render(
    <Router>
      <Header />
    </Router>);
  expect(container).toMatchSnapshot();
});