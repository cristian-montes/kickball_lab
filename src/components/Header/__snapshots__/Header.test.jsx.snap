// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the header with navLinks 1`] = `
<div>
  <div
    class="header"
  >
    <a
      aria-current="page"
      class="navLinks colorClick"
      href="/"
    >
      <p>
        Home
      </p>
    </a>
    <a
      class="navLinks"
      href="/teams"
    >
      <p>
        Teams
      </p>
    </a>
    <a
      class="navLinks"
      href="/players"
    >
      <p>
        Player
      </p>
    </a>
  </div>
</div>
`;
